### DISCUSSION TITLE
Resuming From Snippets

### DISCUSSION THREAD

**Discussion by ThePatches** [nathanhoad/godot_dialogue_manager#544] [Q&A]
I would really like to make use of imported snippets so I can reuse a serious of responses across multiple dialogues, but the documented methods make it unclear how it's supposed to work.

I used the pattern listed

```
~ start
Me: Good Morning. How are you doing?
You: I am fine, but I have some issues.
=>< issues

Me: Oh that seems tragic.

=> END

~ issues
% You: I am sick.
% You: My fingers got broken.
```

According to the documentation, the dialogue should resume after jumping to issues, but it it loops continually back into the jump. Is that not how I am supposed to use this or is the feature wrong? What I actually want to do is something like `=>< banter_snippets/banter_start` jumping into a snippets file which I can reuse-across similar chats.

Thanks in advance,

-- L

**Comment by ThePatches** [comment]
By thinking about it clearly, what's happening is the `=><` operator pushes you to the previous title afterward (which makes sense), but it means you need to use a conditional and a state to jump out of the recursion. Doable, but is less intuitive than what's said in the docs.

**Reply by nathanhoad** [reply]
That was probably a bug which I think has now been fixed if you want to give it another try.

**Reply by ThePatches** [reply]
Thanks! I noticed, but my ADHD brain had gone brrrrr deep into other aspects of the project. I've been using your system to essentially write a visual novel in a weird format, so it's been very helpful.

**Comment by Dagulus** [comment]
As of 18/08/2024 after updating to Godot 4.3 this still seems to be working strangely.

Basically, if you use =>< it seems to execute the jump until it hits END and then loop through the whole of the jump again.
